setup.py

```diff
@@ -65,7 +65,14 @@ def load_requirements(*requirements_paths):
         'workbench',
     ],
     install_requires=load_requirements('requirements.txt'),
-    tests_require=load_requirements('test-requirements.txt'),
+    tests_require=[
```

Comment:
I'm fine with deleting `test-requirements.txt` and putting that info directly into the `setup.py` file, but I'm curious why you want to do so -- especially since you're leaving `requirements.txt` unchanged. What's the difference?